.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+on:
+  schedule:
+    - cron: "0 9 */5 * *"
```

Comment:
Is once a month enough?

Reply:
Is that a proposal? Right now it's every 5 days right?

Reply:
Oh yeah, I misinterpreted this. Every fifth day should be fine